lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -174,25 +174,28 @@ def get(self):
                     apis[path] = urljoin(host, url)
                     templates[path] = template
 
+        endpoints = {
+            "identification": f"Pbench server {self.server_config.COMMIT_ID}",
+            "api": apis,
+            "uri": templates,
+        }
+
         try:
-            oidc_secret = self.server_config.get("authentication", "secret")
-        except NoOptionError:
-            oidc_secret = ""
-        try:
+            secret = self.server_config.get("authentication", "secret")
             client = self.server_config.get("authentication", "client")
             realm = self.server_config.get("authentication", "realm")
             issuer = self.server_config.get("authentication", "server_url")
-            endpoints = {
-                "authentication": {
-                    "client": client,
-                    "realm": realm,
-                    "issuer": issuer,
-                    "secret": oidc_secret,
-                },
-                "identification": f"Pbench server {self.server_config.COMMIT_ID}",
-                "api": apis,
-                "uri": templates,
+        except (NoOptionError, NoSectionError):
+            pass
```

Comment:
I'm inclined not to change all this here, but address this in PR #3114.

I really had hoped we'd have the KeyCloak work in place, so these changes were a minimal way to get around it without ripping it out.

Since this works as is in our staging environment, I'd rather try to address this in another PR so that we don't make further changes to code here that need to be retested.

I'll track this is a follow-up issue.

Reply:
Filed #3143.